src/ui/reducers/app.ts

```diff
@@ -147,9 +144,6 @@ const appSlice = createSlice({
       // Load multiple event types into state at once
       Object.assign(state.events, action.payload);
     },
-    setHoveredLineNumberLocation(state, action: PayloadAction<Location | null>) {
```

Comment:
:question: what's replacing this?

Reply:
It's in the `SourcesContext`
https://github.com/replayio/devtools/blob/1608dc4aa703514272f8c9e7d827f3ee87fcc722/packages/bvaughn-architecture-demo/src/contexts/SourcesContext.tsx#L131-L145